pywincffi/wintypes/__init__.py

```diff
@@ -0,0 +1,113 @@
+"""
+Windows Types Package
+====================
+
+Provides user accessible types corresponding to the respective Windows types
+used across the exposed APIs.
+"""
+
+
+from six import integer_types
+
+from pywincffi.core import typesbase
+from pywincffi.core import dist
+
+
+_FFI, _ = dist.load()
+
+
+# pylint: disable=protected-access
+def wintype_to_cdata(wintype):
+    """
+    :param wintype:
```

Comment:
Could we get a brief description of the function's purpose and maybe a small example?